Docs readmes (#22)
twilio/twilio-voice-react-native-app: README.md

# Twilio Voice React Native Reference

> This project consists of a backend server and a react native app demonstrating best practices for developing a Twilio Programmable Voice application.

Reviewer: ```suggestion
This project consists of a backend server and a React Native app demonstrating best practices for developing a Twilio Programmable Voice application.
```

> This project consists of a backend server and a React Native app demonstrating best practices for developing a Twilio Programmable Voice application.

Reviewer: ```suggestion
This project consists of a backend server and a React Native app that demonstrate best practices for developing a Twilio Programmable Voice application.
```

## Features

* Dialpad for making outgoing calls to a phone number
* Active call screen to interact with and control ongoing calls

### Planned Features

> * Call to a Twilio Client

Reviewer: This should already work. Are we unable to establish client-to-client calls, or do we mean from app to app?

Author: I meant app to app. I'll update.

> <sup>*</sup> Planned for a future release.

Reviewer 2: I think it might make more sense to do a future release section?

Author: Good idea. Maybe something like "Coming soon". Let's see what the whole team thinks as well. @bobiechen-twilio @mhuynh5757

Reviewer: Sounds good, I think a `Planned Features` or `Future Release` section is a good idea.

![dialer](https://user-images.githubusercontent.com/22135968/223575482-53f733b4-c828-4dd1-b100-7cc945a52c9f.jpg)
![active](https://user-images.githubusercontent.com/22135968/223575472-f112ab20-f11c-4f54-9432-3640ddbb0f1c.jpg)

## Prerequisites

> - Understanding of [Twilio Programmable Voice](https://www.twilio.com/docs/voice/sdks)

Reviewer: ```suggestion
- Understanding of [Twilio Programmable Voice](https://www.twilio.com/docs/voice/sdks).
```

> - [React Native Development Environment](https://reactnative.dev/docs/environment-setup)

Reviewer: ```suggestion
- A [React Native Development Environment](https://reactnative.dev/docs/environment-setup).
```
Should we note the version of RN we use and link to those docs, or at least recommend users follow the setup guides for that version?

Author: Good idea. I'll use the previous link you used.

> - Latest LTS version of [NodeJS](https://nodejs.org/en/)

Reviewer: ```suggestion
- Latest LTS version of [NodeJS](https://nodejs.org/en/).
```
